instrumentation/jboss-logmanager-mdc-1.1/javaagent/src/test/groovy/JbossLogmanagerMdcTest.groovy

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import io.opentelemetry.api.trace.Span
+import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
+import org.jboss.logmanager.ExtLogRecord
+import org.jboss.logmanager.Level
+import org.jboss.logmanager.LogContext
+
+import java.util.logging.Handler
+import java.util.logging.LogRecord
+
+class JbossLogmanagerMdcTest extends AgentInstrumentationSpecification {
+  class LogHandler extends Handler {
+    public LinkedList<ExtLogRecord> logRecords
```

Comment:
Prefer using interface, not implementation
```suggestion
    public List<ExtLogRecord> logRecords
```

Reply:
done